web_ui/ui.go

```diff
@@ -507,6 +507,35 @@ func deleteApiToken(ctx *gin.Context) {
 	})
 }
 
+func listApiTokens(ctx *gin.Context) {
+	authOption := token.AuthOption{
+		Sources: []token.TokenSource{token.Cookie},
+		Issuers: []token.TokenIssuer{token.LocalIssuer},
+		Scopes:  []token_scopes.TokenScope{token_scopes.WebUi_Access},
+	}
+	status, ok, err := token.Verify(ctx, authOption)
+	if !ok {
+		log.Warningf("Cannot verify token: %v", err)
```

Comment:
Can you add a bit more context to the warning to indicate more about which function is throwing the error? I worry we have so many "tokens" in Pelican, it won't be obvious what kind of token is failing to verify in these logs.